playbooks/pki.yml

```diff
@@ -143,7 +143,7 @@
   tasks:
     - name: Check for provided host's PKCS12 container
       ansible.builtin.stat:
-        path: "{{ hostvars.localhost.pki_dir }}/certificates/{{ inventory_hostname }}.p12"
+        path: "{{ hostvars.localhost.pki_dir }}/certificates/{{ inventory_hostname | ansible.builtin.hash }}.p12"
```

Comment:
Here we're offering users the possibility to put their own certificates from their own PKI. Having an mapping  inventory_hostname -> file is nice and easy.
I would argue that even if your invnetory hostname has some `/` as a grouping or separator character, it makes a lot of sense that one creates folder structure (that's a container) that match the inventory hostname.

On the other end having to do manual char replace or or compute a hash sounds not very natural to me.